awx/coverage_wsgi.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2015 Ansible, Inc.
+# All Rights Reserved.
+import coverage
+import atexit
+
+import django  # NOQA
+
+from .wsgi import get_wsgi_application
+
+
+"""
+WSGI config for AWX project with code coverage metric collection enabled.
+
+It exposes the WSGI callable as a module-level variable named ``application``.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/dev/howto/deployment/wsgi/
+"""
+global cov
+cov = coverage.Coverage()
+cov.start()
```

Comment:
hmmm this looks like it'll always _start_ recording on deploy/startup, is that right?

is it not "opt-in"?